classes/block_manager.php

```diff
@@ -20,12 +20,12 @@
 
 use core\output\notification;
 use moodle_exception;
-use moodle_page;
 use stdClass;
 use context;
 use context_course;
+use context_system;
```

Comment:
Since 4.2 (see https://tracker.moodle.org/browse/MDL-74936) context_system is an alias of core/context/system. Can you please create a new issue + follow up MR to fix all classes through the code?